caffe2/core/tensor.cc

```diff
@@ -124,6 +124,34 @@ Tensor empty(at::IntList dims, at::TensorOptions options) {
   return tensor;
 }
 
+void reinitialize_(
```

Comment:
probably can benefit from a better name? ReinitializeBuffer or something?

Reply:
Sure

Reply:
I'll change it to ReinitializeTensor